src/main/java/tc/oc/pgm/listeners/PGMListener.java

```diff
@@ -125,6 +132,7 @@ public void removePlayerOnDisconnect(PlayerQuitEvent event) {
     }
 
     match.removePlayer(event.getPlayer());
+    PGM.get().getFlairRegistry().removeFlair(event.getPlayer().getUniqueId());
```

Comment:
`removeFlair` does not seem like a good name for this. FlairRegistry#removePlayer would make alot more sense